__tests__/app.test.js

```diff
@@ -155,7 +162,17 @@ describe("Set of GET Tests", () => {
         .expect(404)
         .then((response) => {
           expect(response.body.msg).toBe(
-            "No Comments Found For This Article..."
+            "No Comments Found For This Article ID..."
+          );
+        });
+    });
+    test("should return a Status Code: 404 for an article ID with no comments yet", () => {
```

Comment:
Is this an error? If the request still worked, it would make sense for this to be a 200 but just return an empty array

Reply:
Also consider 

invalid ID, e.g. api/articles/not-an-id/comments
non existent ID, e.g. api/articles/999/comments